### ISSUE TITLE
Can't log in

### ISSUE THREAD

**Issue by wheeliemow** [blogifierdotnet/Blogifier#131]
Cloned site.
Running in Visual Studio.
Trying to log in with admin default account and demo default account. Site returns to main page and does not log in. 
Seems like the return url is not getting set from the query string.

**Comment by rxtur** [comment]
So login link on front page does not change to "admin"? What happens if you just go to /admin?

**Comment by wheeliemow** [comment]
No the login doesn't change to "admin". If I try and go to the /admin I am redirected to the home page. I traced it and the signinasync is coming back succeeded. But the returnUrl parameter is null.

**Comment by rxtur** [comment]
I suspect this is due to some kind of restrictions you have in your installation/environment. Can you try to [build and run it outside VS](https://github.com/blogifierdotnet/Blogifier/blob/master/docs/Build.md)? Also, does sample site login works for you? (in case it is browser related).

**Comment by wheeliemow** [comment]
That is very possible. I tried it on a different development machine and received a different error. I then went back a couple of versions and everything worked. I actually went back to version 1.5 because I want to incorporate the blog into another MVC site that is not on Core yet. Version 1.5 will work for what I need.

This is a great project! Love the simplicity of everything. 
Keep up the great work!
